resources/config.php

```diff
@@ -1,28 +1,35 @@
 <?php
-// Getting credentials from json file
-$creds = json_decode(file_get_contents("dbcred.json"), true); 
 
+// PHP Error Hide
+// error_reporting(0);
+
+// PHP Error Show
+// ini_set('display_errors', 1);
+// ini_set('display_startup_errors', 1);
+// error_reporting(E_ALL);
+
+// Local site name
+$sitename = 'bookhive.io';
```

Comment:
Lets not hardcode this here. Instead read the values from `/dbcred.json` file. as it is done in https://github.com/JayKandari/bookhive/pull/22/files#diff-b4385cb1277c86f7f86ff5fa6fcfa0f1R16 PR

Reply:
He, I just used in the GitHub resolve conflict feature to resolve the error